app/build.gradle

```diff
@@ -114,6 +117,14 @@ dependencies {
 
     //pager indicator
     implementation 'com.google.accompanist:accompanist-pager-indicators:0.23.1'
+
+    implementation platform('androidx.compose:compose-bom:2022.10.00')
```

Comment:
Why do we use Bill of Materials here? And if we used it then what about the existing compose libraries with the version numbers?

Now, regarding the concern about existing Compose libraries and version numbers, using a BOM doesn't render existing version numbers meaningless. It provides a way to harmonize library versions and ensure compatibility, but it doesn't prevent you from specifying library versions explicitly if needed.

You can still specify a particular version for a library in the project, and the BOM will ensure that the chosen version is compatible with other libraries in the BOM.

Reply:
I understand that, but it is because of when I took clone of the repo, there were a lot of errors regarding compose version, kotlin version and gradle version etc being incompatible with each other because of the code was a bit older.
I had spent enough time fixing this version incompatibility errors and at that time, this was the only fix that seemed fine.

Reply:
Please find the proper alternative or else should update the compatible versions in the separate PR.